org.eclipse.lsp4e/src/org/eclipse/lsp4e/LanguageServerWrapper.java

```diff
@@ -439,20 +322,33 @@ public synchronized void start() throws IOException {
 		}
 	}
 
+	private CompletableFuture<InitializeResult> initServer(final URI rootURI) {
+		String name = "Eclipse IDE"; //$NON-NLS-1$
```

Comment:
I would rather declare name as final and assign the default value in the else branch

Reply:
Used a ternary expression